src/main/java/org/verdictdb/coordinator/ExecutionContext.java

```diff
@@ -89,7 +93,24 @@ public VerdictResultStream streamsql(String query) throws VerdictDBException {
       VerdictResultStream stream = new VerdictResultStreamFromExecutionResultReader(reader, this);
       return stream;
     } else if (queryType.equals(QueryType.scrambling)) {
-      ScramblingCoordinator coordinator = new ScramblingCoordinator(context.getCopiedConnection());
+      CreateScrambleQuery scrambleQuery = generateScrambleQuery(query);
+      ScramblingCoordinator scrambler =
+          new ScramblingCoordinator(context.getCopiedConnection(), scrambleQuery.getNewSchema());
+
+      // Specifying size/ratio of scrambled table is not supported.
+      if (scrambleQuery.getSize() != 1.0) {
+        throw new VerdictDBTypeException(
+            String.format("Scramble size of %f not supported.", scrambleQuery.getSize()));
+      }
+
+      // TODO: store this to our own metadata db later
+      ScrambleMeta meta =
+          scrambler.scramble(
+              scrambleQuery.getOriginalSchema(),
+              scrambleQuery.getOriginalTable(),
+              scrambleQuery.getNewSchema(),
+              scrambleQuery.getNewTable(),
+              scrambleQuery.getMethod()); // dyoon: size is not used atm?
```

Comment:
That's a good point. For now, let's assume that the default value is 100%, and later add a logic to save/read the extra information. For backward compatibility, the missing value can be regarded as 100%.